.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore MD test file
+test.MD
+
+# Ignore node_modules folder
+node_modules
```

Comment:
- [ ] Great job on adding the node_modules file to the .gitignore file. however, the file is still visible in the repository, the reason for this is that you created the file and committed it before adding it to the .gitigone file. to undo this you can run this command `git rm --cached node_modules`. then you can add and commit. with that, the file will not be available in the repository. After this, ou are good to go 😄

Reply:
Hi @BenMukebo, I have done the change you suggested, thanks